src/compiler/diagnosticMessages.json

```diff
@@ -2512,7 +2517,7 @@
         "category": "Message",
         "code": 6084
     },
-    "Enable tracing of the module resolution process.": {
+    "Enable tracing of the name resolution process.": {

```

Comment:
`name` here is either a module name or library name which by itself is not necessary a file name (but it will be resolved to a file name)